app/services/encrypted_key_maker.rb

```diff
@@ -59,8 +59,12 @@ def make_kms(user_access_key)
   end
 
   def unlock_kms(user_access_key, encryption_key)
-    ciphertext = user_access_key.xor(decode(encryption_key))
-    user_access_key.unlock(aws_client.decrypt(ciphertext_blob: ciphertext).plaintext)
+    ciphertext = user_access_key.xor(decode(encryption_key)).sub(KEY_TYPE[:KMS], '')
+    begin
```

Comment:
How do you feel about just adding a top-level rescue?

```ruby
def unlock_kms(...)
  ciphertext = ...
  user_access_key.unlock ...
rescue Aws::KMS::Errors::InvalidCiphertextException
  raise Pii::EncryptionError
end
```

Also, should we copy the message?

```ruby
rescue Aws::KMS::Errors::InvalidCiphertextException => err
  raise Pii::EncryptionError.new(err.message)
```